### ISSUE TITLE
Compatibility with newer versions of RN

### ISSUE THREAD

**Issue by archcorsair** [chaimPaneth/react-native-jw-media-player#127]
With React Native 0.64 released recently I've been working on upgrading my project, however I have run into some dependency related issues and have pinpointed it to this project having very old deps. So old in fact that we need to use `npm install --legacy-peer-deps` when `react-native-jw-media-player` is present.

Have you considered updating the project's dependancies to work with newer releases of RN?

```json
"peerDependencies": {
    "react-native": "^0.40.0",
    "react": "~15.4.0-rc.4"
  }
```
Current releases
`react-native: 0.64`
`react: 17.0.1`

**Comment by KestasVenslauskas** [comment]
Was about to write about the warnings I get:

warning " > react-native-jw-media-player@0.1.53" has incorrect peer dependency "react-native@^0.40.0".
warning " > react-native-jw-media-player@0.1.53" has incorrect peer dependency "react@~15.4.0-rc.4".

Did not upgraded to 0.64 yet but still this has to be addressed.

**Comment by erenkulaksiz** [comment]
Yeah, i got react native version mismatch error while building the example too.

**Comment by chaimPaneth** [comment]
Hi I've upgraded the peer dependencies in the latest version but they might mismatch with what you have installed, for now you can use --force on install.

The library just needs any version of react and react-native installed to work, so the exact match doesn't matter
